README.md

```diff
@@ -164,7 +164,7 @@ See the [contributing guide](CONTRIBUTING.md) on how you can help improve Zed!
 
 ## Join the Community
 
-Join our [Public Slack](https://www.brimdata.io/join-slack/) workspace for announcements, Q&A, and to trade tips!
+Join our [public Slack](https://www.brimdata.io/join-slack/) workspace for announcements, Q&A, and to trade tips!
```

Comment:
I think I was the one that incorrectly capitalized "Public" back in the day, so I'm just fixing my own error here. :wink: